Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -485,10 +496,24 @@ - (void)updateBarButtonItems
     }
 
     //  Settings button.
-    //
-    // TODO: Theme
-    UIImage *image = [UIImage imageNamed:@"button_settings_white"];
-    UIBarButtonItem *settingsButton = [[UIBarButtonItem alloc] initWithImage:image style:UIBarButtonItemStylePlain target:self action:@selector(settingsButtonPressed:)];
+    const NSUInteger kAvatarSize = 28;
+    // TODO: Replace this icon.
```

Comment:
re: this todo, do you mean instead of using the profile_avatar_default needs to change? Is that already on Myles's radar?

Reply:
Myles has clarified the design.  I'll address in a follow-up PR.